pkg/client/cli/cmd_status.go

```diff
@@ -148,6 +150,12 @@ func BasicGetStatusInfo(ctx context.Context) (ioutil.WriterTos, error) {
 				Client: icept.Spec.Client,
 			})
 		}
+		if ms := status.ManagerStatus; ms != nil {
```

Comment:
Will `ManagerStatus` be set even when connecting to an older traffic-manager?

Reply:
Yeah this was coming in as nil if the tm was old or not connected; no way to tell the difference between both cases.
Ended up propagating the tm version all the way to the cli where it checks the version before printing the new status fields.